modules/nf-core/tcoffee/regressive/meta.yml

```diff
@@ -0,0 +1,80 @@
+name: "tcoffee_regressive"
+description: Aligns sequences using T_COFFEE
```

Comment:
Make it more descriptive, this is a specific algorithm

Reply:
ok @itrujnara, done! Thanks for the review